rest/.jvm/src/main/scala/io/udash/rest/openapi/OpenApiServlet.scala

```diff
@@ -14,10 +14,7 @@ object OpenApiServlet {
     security: List[SecurityRequirement] = Nil,
     tags: List[Tag] = Nil,
     externalDocs: OptArg[ExternalDocumentation] = OptArg.Empty
-  ): OpenApiServlet = new OpenApiServlet {
-    protected def render(request: HttpServletRequest): OpenApi =
-      implicitly[OpenApiMetadata[RestApi]].openapi(info, components, servers, security, tags, externalDocs)
-  }
+  ): OpenApiServlet = _ => implicitly[OpenApiMetadata[RestApi]].openapi(info, components, servers, security, tags, externalDocs)
```

Comment:
`OpenApiServlet` is hardly a "functional interface", IMO here SAM lambda hurts readability.